### DISCUSSION TITLE
Auto correct

### DISCUSSION THREAD

**Discussion by vishaltanna** [flyerhq/react-native-chat-ui#65] [Ideas / Feature Requests] [answered]
Hello All,

is auto correct option available for message input?

Can anybody help me how I can achieve this?

**Comment by demchenkoalex** [comment, marked as answer]
You have access to `textInputProps`, which is a usual `TextInput` props, including auto correct.

**Reply by vishaltanna** [reply]
@demchenkoalex Thanks. I will check it out for this.
